pkg/apis/externaldns/validation/validation.go

```diff
@@ -52,5 +52,18 @@ func ValidateConfig(cfg *externaldns.Config) error {
 			return errors.New("no Infoblox WAPI password specified")
 		}
 	}
+
+	if cfg.Provider == "dyn" {
+		if cfg.DynUsername == "" {
+			return errors.New("no Dyn username specified")
+		}
+		if cfg.DynCustomerName == "" {
+			return errors.New("no Dyn customer name specified")
+		}
+
+		if cfg.DynMinTTLSeconds < 0 {
+			return errors.New("TTL specified for Dyn is negative")
+		}
+	}
```

Comment:
@jvassev could you add tests for them in `validation_test.go` as well?

Reply:
Added the test, also another one testing the change from my previous commit.